bootstrap/sql/com.mysql.cj.jdbc.Driver/v009__create_db_connection_info.sql

```diff
@@ -60,3 +60,16 @@ CREATE TABLE IF NOT EXISTS test_connection_definition (
     PRIMARY KEY (id),
     UNIQUE (name)
 );
+
+CREATE TABLE IF NOT EXISTS operations_workflow (
```

Comment:
shall we use automation workflow here. We want to go into automations post 1.0 release. We should be using the same workflows entities

Reply:
thanks, will rename to `automations_workflow` & update API tags